openshift-ci/run_frrk8s_e2e.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/bash
+set -ex
+
+FRRK8S_DIR="$(dirname $(readlink -f $0))/../../frr"
+
+KUBECONFIG=$(readlink -f ../../ocp/ostest/auth/kubeconfig)
+pushd $FRRK8S_DIR
+
+SKIP="Leaked.*advertising\|receive.*ips.*from.*some\|VRF.*Advertise.*a.*subset.*of.*ips"
+SKIP="$SKIP\|should.*block.*always.*block.*cidr"
+
+if [[ "$BGP_TYPE" == "frr-k8s" ]]; then
+  SKIP="$SKIP\|metrics"  # because when running as a metallb pod the metrics are overridden.
+fi
+
+FRRK8S_NAMESPACE="metallb-system"
+if [[ "$BGP_TYPE" == "frr-k8s-cno" ]]; then
+  FRRK8S_NAMESPACE="openshift-frr-k8s"
+fi
+
+
+
+if [ "${IP_STACK}" = "v4" ]; then
+	SKIP="$SKIP\|IPV6\|DUALSTACK"
+	export PROVISIONING_HOST_EXTERNAL_IPV4=${PROVISIONING_HOST_EXTERNAL_IP}
+	export PROVISIONING_HOST_EXTERNAL_IPV6=1111:1:1::1
+elif [ "${IP_STACK}" = "v6" ]; then
+	SKIP="$SKIP\|IPV4\|DUALSTACK"
+	export PROVISIONING_HOST_EXTERNAL_IPV6=${PROVISIONING_HOST_EXTERNAL_IP}
+	export PROVISIONING_HOST_EXTERNAL_IPV4=1.1.1.1
+elif [ "${IP_STACK}" = "v4v6" ]; then
+	SKIP="$SKIP\|IPV6"
+	export PROVISIONING_HOST_EXTERNAL_IPV4=${PROVISIONING_HOST_EXTERNAL_IP}
+	export PROVISIONING_HOST_EXTERNAL_IPV6=1111:1:1::1
+fi
+echo "Skipping ${SKIP}"
+
+## In order to make the VRF leaking tests work we need to
+pods=$(oc get pods -l "app=frr-k8s" -n $FRRK8S_NAMESPACE -o jsonpath='{.items[*].metadata.name}')
+
+echo "creating vrfs in pods $pods"
+for pod in $pods; do
+  echo "creating vrf in pod $pod"
+  oc exec $pod -n $FRRK8S_NAMESPACE -c frr -- ip link add red type vrf table 1100
+  oc exec $pod -n $FRRK8S_NAMESPACE -c frr -- ip link set red up
+done
+
+export TEST_ARGS="--frr-k8s-namespace=$FRRK8S_NAMESPACE --kubeconfig=$KUBECONFIG --prometheus-namespace=openshift-monitoring --report-path=$REPORTER_PATH"
+
+export RUN_FRR_CONTAINER_ON_HOST_NETWORK="true"
```

Comment:
question: is there a reason why that is ENV and not being a arg addeded in TEST_ARGS?

Reply:
because there's no such argument, only the env variable is used. You can check in the e2e test code: https://github.com/metallb/frr-k8s/blob/3cafec5866f8a6114004bfebda64c6dad8a5e277/e2etests/e2etest_suite_test.go#L46

Reply:
sure, the question was more curiousity why historically was not added as arg and ENV was prefered.